build.js

```diff
@@ -4,31 +4,33 @@ import { fileURLToPath } from 'url';
 
 import { build } from 'esbuild';
 
+const [, , logLevel = 'info'] = process.argv;
 const pkg = JSON.parse(await readFile(new URL('package.json', import.meta.url)));
 
 await build({
   entryPoints: ['src/index.ts', 'src/tailwindcss.worker.ts'],
   bundle: true,
   external: Object.keys({ ...pkg.dependencies, ...pkg.peerDependencies }),
-  logLevel: 'info',
+  logLevel,
   outdir: '.',
   sourcemap: true,
   format: 'esm',
   target: ['es2020'],
   define: {
     'process.env.DEBUG': 'undefined',
+    'process.env.JEST_WORKER_ID': 'undefined',
     __dirname: '"/"',
   },
   plugins: [
     {
       name: 'alias',
-      async setup({ onResolve, resolve }) {
+      async setup({ onLoad, onResolve, resolve }) {
         const stubFiles = await readdir('src/stubs', { withFileTypes: true });
         // These packages are imported, but can be stubbed.
         const stubNames = stubFiles
           .filter((file) => file.isFile())
           .map((file) => parse(file.name).name);
-        onResolve({ filter: new RegExp(`^(${stubNames.join('|')})$`) }, ({ path }) => ({
+        onResolve({ filter: new RegExp(`^(${stubNames.join('|')})(\\/|$)`) }, ({ path }) => ({
```

Comment:
since this was based on the other pr, this needs to be reverted here too https://github.com/remcohaszing/monaco-tailwindcss/pull/41#discussion_r851600965